Makefile

```diff
@@ -124,6 +124,12 @@ agent-rest:
 	@mkdir -p ./build/bin
 	@cd ${ARIES_AGENT_REST_PATH} && go build -o ../../build/bin/aries-agent-rest main.go
 
+.PHONY: agent-rest-interop
```

Comment:
we should make these targets specific to acapy-interop.

Reply:
@troyronda could you clarify what you mean?
rename the target, or remove the target and have the flagged build only built in interop tests?

Reply:
similar to @llorllale comment earlier, interop is too generic.